CONTRIBUTING.rst

```diff
@@ -174,3 +174,22 @@ Then you are done.
 
 If the pull request has been opened for the ``f25-devel`` branch, then you also need to merge the ``f25-devel``
 branch back to ``master`` once you merge your pull request (see "Merging the Fedora devel branch back to the master branch" above).
+
+Pure community features
+-----------------------
+
+The pure community features are features which are part of the Anaconda code base but they are maintained and extended mainly by the community. These features are not a priority for the Anaconda project.
+
+In case of issues in pure community features, the Anaconda team will provide only sanity checking. It is the responsibility of the community (maintainers of the feature) to provide fix for the issue. If the issue will have bigger impact on other parts of the Anaconda project or if it will block a release or another priority feature and the fix won't be provided in a reasonable time the Anaconda team have the rights to remove or disable this feature from the Anaconda code base.
```

Comment:
`the Anaconda team have the rights to remove or disable this feature`
This seams a bit weird to me, I suggest using this:
`the Anaconda team reserves the right to remove or disable this feature`

Reply:
Fixed.